common/log/slog.go

```diff
@@ -44,8 +44,17 @@ type handler struct {
 
 var _ slog.Handler = (*handler)(nil)
 
+// An interface that allows extracting an underlying slog logger.
+type SLogWrapper interface {
```

Comment:
I don't see any usage of this SLogWrapper.

Reply:
Users that provide a custom logger may want to reference this interface.

I made a mistake and didn't use the Go docstring convention. Fixing.